functests/0_config/config.go

```diff
@@ -37,6 +40,22 @@ import (
 
 var _ = Describe("[performance][config] Performance configuration", func() {
 
+	It("Should run performance profile pod on a master node", func() {
+		pod, err := pods.GetPerformanceProfilePod()
+		Expect(err).ToNot(HaveOccurred(), "Failed to find the PAO pod")
+
+		masterNodes, err := nodes.GetByRole(testutils.RoleMaster)
+		Expect(err).ToNot(HaveOccurred(), "Failed to query the master nodes")
+		for _, node := range masterNodes {
```

Comment:
alternatively you can get the node by the name and check its role (both ways work)

Reply:
Yeah, I'd do like @yanirq wrote, but both works so I don't really mind.

Reply:
I am missing something, the code does queries the node "name" from the pod, iterates over the master nodes to see if is one of the master nodes.
The other way would be to go over all nodes and find the node with the right name, then query its labels to see is if master.

The above seems more concise/straight forward... but I can rewrite is you see an issue

Reply:
The alternative is to get node by name (using server-side filtering ofc), getting the node from pod Spec, and finally checking the node you got has the master labels (thus, it's a master node). Both approaches work, and I don't mind, because masters are supposed to be few in number anyway.

Reply:
got it, thanks